### DISCUSSION TITLE
Jupyter Notebook-The kernel appears to have died. It will restart automatically

### DISCUSSION THREAD

**Discussion by kalyan678** [ourownstory/neural_prophet#393] [Q&A - get help using NeuralProphet] [answered]
Hello,
I’m writing to report an error when running Neural Prophet in python Jupyter notebook, and ask for help on fixing this error. It keeps shutting down my kernel resulting in an error message “The kernel appears to have died. It will restart automatically.” 
Please note that I have installed Neural Prophet live through -pip install in my anaconda package #management. I tried my best to figure out the issue but I literally failed here. Could you please help me?
![Error](https://user-images.githubusercontent.com/46974898/135104508-55fd5246-fd0e-4f3c-9822-9ee72ce3e792.png)
![ss_2](https://user-images.githubusercontent.com/46974898/135104571-c7d211f1-eb0a-4c49-84c0-949f2d04e8bb.png)

Thanks a lot for your support.

**Comment by ourownstory** [comment]
Hi, I am sorry, I do not know what this could be. Have you tried using venv and pip  instead of conda?
Does this happen on both non-live and live version?

**Comment by kalyan678** [comment]
Hello,

Thanks for your response.
I somehow missed this notification. Apologies for replying late.  I tried on conda and the issue happened with both live & non live versions. May be I will try on venv and see how it works.

**Comment by hongxuzhang** [comment]
I have similar problem with using jupyter running the test code below... it just stuck at first epoch.
How much CPU and memory are needed for this to run? 

![image](https://user-images.githubusercontent.com/26710689/145161182-459d5d1e-c902-4015-aa25-959db5445d0a.png)

**Comment by hongxuzhang** [comment, marked as answer]
I downgrade my numpy version and somehow the problem is solved!

**Reply by Gayatri2703** [reply]
> I downgrade my numpy version and somehow the problem is solved!

For me, it was actually uninstalling and install numpy that worked! Thanks @hongxuzhang

**Reply by ourownstory** [reply]
Thank you @hongxuzhang  and @Gayatri2703  for sharing what solved the issue for you!
This is odd - please let me know if you have any further ideas what exactly might have been the issue.

**Reply by stuartvkerr** [reply]
which version of numpy did you downgrade to?

**Reply by theatifwaheed** [reply]
@hongxuzhang which numpy version?

**Reply by ksraghuram** [reply]
Numpy version 1.26.4 worked for me. I downgraded from 2.1.2
